OnShift/SendBird-JavaScript
# Misc work

# Things Done
- button should now 'scroll' with pages that take up more than a single page length
- updated npm meta stuff
- push notifications got me down so I did this instead...

# How To Test
- load up the widget with engage
- see how it scrolls and is generally responsive
- should still publish and all that
